src/connectors.rs

```diff
@@ -1176,6 +1177,7 @@ pub(crate) fn builtin_connector_types() -> Vec<Box<dyn ConnectorBuilder + 'stati
         Box::new(impls::otel::client::Builder::default()),
         Box::new(impls::otel::server::Builder::default()),
         Box::new(impls::gbq::writer::Builder::default()),
+        Box::new(impls::gpubsub::gsub::Builder::default()),
```

Comment:
would it make sense to encapsulate that into a `consumer` sub module to mirror the kafka one?

Reply:
It's already in the `gsub` submodule (and there will be `gpub` in another PR), but I can change the nomenclature to `consumer`/`producer`, I don't really have a strong opinion here, though I like `gsub`/`gpub` as connector names better than `gpubsub-consumer`/`gpubsub-producer`

Reply:
I think for recognizability and pettern repetition it's better to have a consumer/producer.

I've never heard to it referred to as `gpub` only as google pub/sub (gpubsub) so throwing in `gpub` and `gsub` will make discovery for users harder.

For the `producer/consumer` it's nice to have repeatable patterns for approximately equal services, pubsub, kafka, (whatever other message bus we add) if they all have a producer and consumer people can get used to that pattern, same with `client` and `server` or `reader` and `writer`